### PR TITLE
 Setup mobile first project

### PR DESCRIPTION
**Pull request by Dorcas126**
In this project i worked on a mobile first 
I used HTML and 
CSS .

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        <nav class="navlist">
+            <img src="img/Union.png" alt="navLogo">
+        </nav>
+    </header>
+
+    <main class="mainContent">
+        <section class="Content">
+            <h2>I'm Dorcas 
+                Glad to see you!
+            </h2>
```
Comment (abdulhamiid) [inline index.html]: - [ ] Please kindly make the main title `I'm Dorcas ...` an `h1` tag. 🛠👍

- [ ] If you want that google (and other search engines) to know about a specific topic, in this case, you, we can use an h1 tag. 

- [ ] You can [check this link](https://www.reliablesoft.net/h1-tag/) for more info. 🤗

File: index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="img/Vector (10).png" alt="gmail image">
+                        </a>
+                    </li>
+                </ul>
+                <ul>
+                    <li>
+                        <a href="#">
```
Comment (abdulhamiid) [inline index.html]: According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md). the list of icons uses a `ul` tag to group them, and a `li` per icon; each icon has a link (`a` tag) 

i.e all the list items should be grouped inside a single `ul` tag 🤓👍

Example

```HTML
 <ul>
      <li>
          <a href="#">
              <img src="img/Vector (10).png" alt="gmail image">
          </a>
      </li>
      <li>
          . . .
      </li>
      <li>
          . . .
      </li>
  </ul>
```

Kindly fix that 🛠💪

File: README.md [hunk 3]
```diff
@@ -1 +1,174 @@
...
+
+## 👥 Authors 
+
+👤 **Author1**
+
+- GitHub: [@Dorcas126](https://github.com/Dorcas126)
+- Twitter: [@Dorcasuw](https://twitter.com/Dorcasuwi)
+- LinkedIn: [Uwimbabazi Dorcas](www.linkedin.com/in/
+uwimbabazi-dorcas-956a5a226)
+
```
Comment (abdulhamiid) [inline README.md]: Please kindly fix the issue with the link to your `LinkedIn` profile. it is not working correctly. 🛠💪

- It looks like there is a break in between the `url` whereas everything is supposed to be on a single line. 👀

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Dorcas126 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ UI design
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
### Hello @Dorcas126 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- No linter errors :heavy_check_mark: 
- You implemented all suggested changes :heavy_check_mark: 
 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**